sql/parse/parse.go

```diff
@@ -63,6 +64,11 @@ func Parse(ctx *sql.Context, query string) (sql.Node, error) {
 	}
 
 	lowerQuery := strings.ToLower(s)
+	if showWarningsRegex.MatchString(lowerQuery) {
```

Comment:
should we do this on analyzer phase using a rule?

Reply:
We can do it (hopefully no warnings will be missed), by adding a new rule to
`analyzer.OnceBeforeDefault`. Is it a right place?

Reply:
I think it should be on `OnceAfterAll`.

Reply:
ok, it also will work:+1